docs/pages/guides/upgrading/1.0.mdx

```diff
@@ -0,0 +1,161 @@
+
+### Update the authorize option [#authorize]
+
+We have updated the `authorize` option to make `userId` mandatory. This change
+ties into our new [pricing model](https://liveblocks.io/pricing), which is based
+on Monthly Active Users instead of connections. We use `userId` to track MAU
+associated with a Liveblocks account.
+
+If you do not use the `authorize` option, a cookie will be set on the client to
+track the user. This cookie will be used to track MAU associated with a
+Liveblocks account and will expire after 30 days.
+
+### Webhooks Enhancements [#webhooks]
+
+We have added two new events to our Webhook functionality: `RoomCreatedEvent`
+and `RoomDeletedEvent`. We have also added a `WebhookHandler` class to make it
+easier to verify event requests from Liveblock's Webhook functionality. It also
+provides fully typed `WebhookEvents.`
+
+```ts
+import { WebhookHandler } from "@liveblocks/node";
+
+const webhookHandler = new WebhookHandler(process.env.SECRET);
+
+const event = webhookHandler.verifyRequest({
+  headers: req.headers,
+  rawBody: req.body,
+});
+```
+
+Verification of events is critical to ensure that the events are coming from
+Liveblocks and not from a malicious source. Check out our
+[Webhooks guide](https://liveblocks.io/docs/guides/webhooks) for more details.
+
+## Changes for all liveblocks packages [#all]
+
+When upgrading to 1.0, the primary breaking change to consider is updating the
+`authorize` option to include the now-mandatory `userId` field if you use
+`@liveblocks/node`. This change reflects our new pricing model based on Monthly
+Active Users instead of connections.
+
+### Step 1: Update your liveblocks packages [#step1]
+
+```bash
+npm update @liveblocks/client@1.0 @liveblocks/react@1.0 @liveblocks/node@1.0
+```
+
+### Step 2: Update your authorize option [#step2]
+
+```ts highlight="20"
+import { authorize } from "@liveblocks/node";
+
+// Replace this key with your secret key provided at
+// https://liveblocks.io/dashboard/projects/{projectId}/apikeys
+const secret = "sk_prod_xxxxxxxxxxxxxxxxxxxxxxxx";
+
+export default async function auth(req, res) {
+  /**
+   * Implement your own security here.
+   *
+   * It's your responsibility to ensure that the caller of this endpoint
+   * is a valid user by validating the cookies or authentication headers
+   * and that it has access to the requested room.
+   */
+  const room = req.body.room;
+  const response = await authorize({
+    room,
+    secret,
+    // Corresponds to the UserMeta[id] type defined in liveblocks.config.ts
+    userId: "123", // Required
+    groupIds: ["456"], // Optional
+    userInfo: {
+      // Optional, corresponds to the UserMeta[info] type defined in liveblocks.config.ts
+      name: "Ada Lovelace",
+      color: "red",
+    },
+  });
+  return res.status(response.status).end(response.body);
+}
+```
+
+That’s it!
+
+If you have issues with these new patterns and need help, please
+[let us know](/contact)! We’re here to help!
```

Comment:
```suggestion
If you have issues with these new patterns and need help, please let us know [by email](mailto:support@liveblocks.io), or by joining our [Discord community](/discord)! We’re here to help!
```

The `contact` page is too generic, and we're missing an occasion to make people join our Discord server